examples/olsconfig.yaml

```diff
@@ -25,6 +25,7 @@ ols_config:
   default_model: ibm/granite-13b-chat-v2
 dev_config:
   enable_dev_ui: true
-  # llm_temperature_override: 0
+  # llm_params:
+  #   temperature_override: 0
```

Comment:
the reason i implemented this as i originally did is because I anticipate us potentially exposing all llm parameters as part of the model-specific config that even user can provide (or at least that we hardcode into the ols config).

this is ok for now, but will need to be revisited as we evaluate what llm configuration needs to be:
1) controlled on a per model basis, not a global basis
2) controlled by the user vs being hardcoded into OLS